src/lib/redux-store/explore/explore-apis.ts

```diff
@@ -31,18 +37,80 @@ const updateExploreWrapper = (dispatch, metricsDefId: string) => {
 };
 
 /**
- * Initialises explore with dimensions and measures.
- * Selected measures is initialised with all measures in the metrics definition.
+ * Syncs explore with updated measures and dimensions.
+ * If a MetricsExplore entity is not present then a new one is created.
  * It then calls {@link updateExploreWrapper} to update explore.
  */
-export const initAndUpdateExplore = (
+export const syncExplore = (
   dispatch,
   metricsDefId: string,
+  metricsDef: MetricsExploreEntity,
```

Comment:
why is this called `metricsDef`? That doesn't seem to match `MetricsExploreEntity`